lib/std/start.zig

```diff
@@ -147,6 +147,17 @@ fn _start() callconv(.Naked) noreturn {
                 : "r0"
             );
         },
+        .sparcv9 => {
+            // On sparc64, the stack pointer register points to a place
```

Comment:
The comment is a bit verbose, if you're touching this code you'd better know about the SPARC ABI, no?

I'd suggest something short and straight to the point like: `The argv is located after a register window (16 registers), plus the stack bias.```